TELCODOCS-2300: Clarifying network config limitations for IBI (#93143)
openshift/openshift-docs: modules/ibi-image-based-install-cluster-guide.adoc

> * If your cluster deployment at the edge of the network requires a proxy configuration, you must create a seed image from a seed cluster featuring a proxy configuration. The proxy configurations do not have to match.

Reviewer: This is also out of scope but perhaps more worthwhile to address in this PR if you can, but the two `==` headings above this line do not have any IDs on them, which they need to have (and with a `_{context}` at the end)

> * The `clusterNetwork` and `serviceNetwork` network configurations in the seed cluster persist to the deployed cluster. The Lifecycle Agent embeds these settings in the seed image and they cannot be changed later in the image-based installation and deployment process.

Reviewer 2: This is a nit, but you might want to reword the second sentence slightly to use the active voice and to make the structure ("you...") parallel with the previous and next list item.

```suggestion
* The `clusterNetwork` and `serviceNetwork` network configurations in the seed cluster persist to the deployed cluster. The Lifecycle Agent embeds these settings in the seed image. You cannot change these settings later in the image-based installation and deployment process.
```

Reviewer: Link to an ISG rule about active voice for user focus, for anyone interested: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#voice

> * Set the `clusterNetwork` and `serviceNetwork` network configurations in the seed cluster. The Lifecycle Agent embeds these settings in the seed image and they persist to the deployed cluster. You cannot change these network configurations later in the image-based installation and deployment process.

Reviewer 3: The user doesn't need to explicitly set these values, they just need to be aware that the configuration present on the seed cluster persists to the deployed cluster and cannot be changed later.
Consider:
The clusterNetwork and serviceNetwork network configurations on the seed cluster persist to the deployed cluster. The Lifecycle Agent embeds these settings in the seed image, and they cannot be changed later in the image-based installation and deployment process.
